weapon-config.inc

```diff
@@ -4520,6 +4520,18 @@ static ProcessDamage(&playerid, &issuerid, &Float:amount, &weaponid, &bodypart,
 			}
 		}
 	}
+	
+	// Check deagle damage
+	if (weaponid == WEAPON_CHAINSAW) {
```

Comment:
Anyways, I'll do some more testing today because this is an embarrassment and I brought it onto myself all on my own.